test/deploy/kubernetes/template-config/cluster_redefined.yaml

```diff
@@ -2,6 +2,11 @@ cluster:
   name: cortx-cluster
   id: 0007ec45379e36d9fa089a3d615c32a3
   node_types:
+    - name: client_node        # Optional
```

Comment:
these changes needs to be reflected in conf/cluster.yaml.sample as well.

Reply:
Created sample file for cluster-redefine.yaml.